iree/samples/variables_and_state/main.c

```diff
@@ -46,16 +46,15 @@ iree_status_t counter_set_value(iree_runtime_session_t* session,
       session, iree_make_cstring_view("module.set_value"), &call));
 
   iree_hal_buffer_view_t* arg0 = NULL;
-  static const iree_hal_dim_t arg0_shape[1] = {1};
   int arg0_data[1] = {new_value};
 
   // TODO(scotttodd): use iree_hal_buffer_view_wrap_or_clone_heap_buffer
   //   * debugging some apparent memory corruption with the stack-local value
```

Comment:
I tried to address this while I was in the area, but still ran into issues that need more debugging.